lightning/backend/local.go

```diff
@@ -1245,7 +1260,6 @@ func (local *local) CleanupEngine(ctx context.Context, engineUUID uuid.UUID) err
 		if err != nil {
 			return err
 		}
-		local.engines.Delete(engineUUID)
```

Comment:
this line could be kept since `ResetEngine` no longer calling `CleanupEngine`

Reply:
done